### PR TITLE
[strings] Fixed Portuguese exit strings

### PR DESCRIPTION
**Pull request by matheusgomesms**
Quick fix: fixed Portuguese exit (entrance/door) strings.

Although PT and PT-BR translations are the same, since it seems OM is moving translations to Weblate, I preferred to have it duplicated. Let's see how it works in the future.

### CODE REVIEW

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -11343,8 +11343,8 @@
     nb = Utgang
     nl = Ga naar
     pl = Wyjście
-    pt = Sai
-    pt-BR = Sair
+    pt = Saída
+    pt-BR = Saída
```
Comment (biodranik) [inline data/strings/types_strings.txt:11347]: There is no need to add a duplicate.

**Review by biodranik — COMMENTED** [review]
Moving to weblate is not approved yet, the quality of translations there, unfortunately, is not great, and it may seriously complicate development and adding translation strings by collaborators.

**Comment by matheusgomesms** [comment]
I'm talking about #4515, don't how it will be with same strings as EN, same strings related to PT/PT-BR and ES/ES-MX for example... Even though, fixing here now!

File: data/strings/types_strings.txt (hunk 1 above)
Comment (matheusgomesms) [reply data/strings/types_strings.txt]: Removed.

**Review by biodranik — APPROVED** [review]
Obrigado!

**Pull request merged**